client/validate.go

```diff
@@ -36,10 +36,12 @@ type ValidateApplyOptions struct {
 
 // ValidationSetResult holds information about a single validation set.
 type ValidationSetResult struct {
-	ValidationSet string `json:"validation-set,omitempty"`
-	Mode          string `json:"mode"`
-	Sequence      int    `json:"sequence,omitempty"`
-	Valid         bool   `json:"valid"`
+	AccountID string `json:"account-id"`
```

Comment:
maybe a comment before this like

// set sequence key and optional pinned-at (=)